docs/_documentations/installoncloud.md

```diff
@@ -138,18 +138,13 @@ Go to **View**>**Find Command…**>**Codewind: Build**.
 ## Updating the version
 Restart the Codewind workspace in Che. Che automatically pulls the newest version of Codewind and the Theia extension.
 
-## Adding additional rules to support Codewind ODO extension
-The Codewind ODO extension needs to add additional rules for accessing OpenShift resources:
-1. In your home directory, run the following command to clone the Codewind ODO extension repository:
-```
-git clone https://github.com/eclipse/codewind-odo-extension
-```
-2. Login to your OpenShift/OKD cluster
-3. Go into `~/codewind-odo-extension/odo-RBAC` then run the following commands to add additional rules:
-```
-kubectl apply -f codewind-odoclusterrole.yaml
-kubectl apply -f codewind-odoclusterrolebinding.yaml
-```
+# Codewind OpenShift Do (odo) extension 
+See the [`codewind-odo-extension` repository](https://github.com/eclipse/codewind-odo-extension) for the extension to Codewind that provides support for odo projects.
 
-# Codewind OpenShift Do (ODO) extension 
-Extension to Codewind providing support for OpenShift projects: https://github.com/eclipse/codewind-odo-extension
\ No newline at end of file
+## Adding rules to support the Codewind odo extension
```

Comment:
Notice you update the header from `Adding additional rules to support Codewind ODO extension` to `Adding rules to support the Codewind odo extension`, I think the clickable link shouldn't work since the reference `#adding-additional-rules-to-support-codewind-odo-extension` is here: https://github.com/eclipse/codewind-docs/pull/196/files#diff-6e005c6b436a3683f3590defa6715fc5R22. You can use VS Code Markdown Preview to verify if the link works